autogen/README.md

```diff
@@ -13,3 +13,13 @@ Binary package
 ## Usage
 
 *TODO: Add instructions for usage*
+
+## Testing
```

Comment:
Please remove this section in favor of #2678

Reply:
Thanks!